### ISSUE TITLE
bang wong et al color map? 

### ISSUE THREAD

**Issue by jgunstone** [tlambert03/cmap#71]
awesome repo - nice keep colors separate from plotting. 

I use the [Bang Wong color scheme](https://www.nature.com/articles/nmeth.1618) and it would be great to retrieve it from cmap...

described here:
https://davidmathlogic.com/colorblind/#%23000000-%23E69F00-%2356B4E9-%23009E73-%23F0E442-%230072B2-%23D55E00-%23CC79A7

**Comment by tlambert03** [comment]
thanks @jgunstone.

Could I entice you to contribute? 
https://cmap-docs.readthedocs.io/en/stable/faq/#how-can-i-add-a-new-colormap

we can create a new `wong` namespace for it

**Comment by beskep** [comment]
It seems to be the same as Okabe-Ito I think?

- https://cmap-docs.readthedocs.io/en/latest/catalog/qualitative/okabeito:okabeito/
- https://jfly.uni-koeln.de/color/

**Comment by jgunstone** [comment]
hey - good spot - I think you are right: 

<img width="553" alt="image" src="https://github.com/user-attachments/assets/1c7ba1ff-866c-447d-a175-ebc7ee99b178">

top: https://cmap-docs.readthedocs.io/en/latest/catalog/qualitative/okabeito:okabeito/
bottom: https://davidmathlogic.com/colorblind/#%23000000-%23E69F00-%2356B4E9-%23009E73-%23F0E442-%230072B2-%23D55E00-%23CC79A7

**Comment by tlambert03** [comment]
indeed,  looks like Wong posted a correction to the original article, acknowledging that he failed to reference okabe ito https://www-nature-com.ezp-prod1.hul.harvard.edu/articles/s41592-023-01974-0

pretty glaring omission actually 😬

i will add a note to the okabe-ito blurb, so that anyone searching for bang wong may find it

**Comment by tlambert03** [comment]
thanks @jgunstone, and thanks @beskep for the catch
